modules/serverless-kafka-sasl.adoc

```diff
@@ -0,0 +1,83 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-configuring-sasl-authentication-against-an-apache-kafka_{context}"]
+== Configuring SASL authentication
```

Comment:
Module titles should always be `= Title of module`, vs `== Title of module`. You can adjust the heading level in the include statement in the assembly. 👍